src/App.tsx

```diff
@@ -1,14 +1,62 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './App.scss';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedOption, setSelectedOption] = useState('all');
+  const [title, setTitle] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [currentTodo, setCurrentTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    setLoading(true);
+    getTodos()
+      .then(todo => {
+        setTodos(todo);
+      })
+      .finally(() => setLoading(false));
+  }, []);
+
+  const filterTodos = (option: string, titl: string): Todo[] => {
+    let filteredTodos = todos;
+
+    switch (option) {
+      case 'active':
+        filteredTodos = todos.filter(todo => !todo.completed);
+        break;
+
+      case 'completed':
```

Comment:
Use a enum for 'active' and 'completed' 
<img width="355" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/64750958/9e2c97b3-156a-48a6-beb5-47939083550b">

Reply:
Consider creating an enum as mentioned by the previous reviewer. You can avoid typos and use code editor suggestions by using it.